metrics/conflict.py

```diff
@@ -227,16 +234,8 @@ def get_conflict_score_per_editor(self):
         joined = confs_n.join(confs_ed).join(actions)
 
         # calculate the score of the editor dividing conflicts / actions
-        joined['conflict_score'] = joined['conflict'] / joined['action']
+        joined['conflict_score_ed'] = joined['conflict'] / joined['action']
         joined['conflict_ratio'] = joined['conflict_n'] / joined['action']
 
         # return the result sorted in descending order
-        return joined.sort_values('conflict_score', ascending=False)
-
-    # c_t = np.log(3600) / (
-    #     np.log(
-    #         dups_dated.loc[conflicts,['token_id','time_diff']].groupby(
-    #             'token_id').sum().astype('timedelta64[s]') + 2
-    #     ))
-
-    # c_t.sum()
+        return joined.sort_values('conflict', ascending=False)
```

Comment:
here you are sorting by `conflict` and before was sorted by the `conflict_score`. I am not sure which does make more sense, or if they are even differrent, ?